chains/arbitrum/config/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use rosetta_core::{
+    crypto::{address::AddressFormat, Algorithm},
+    BlockchainConfig, NodeUri,
+};
+
+/// Retrieve the [`BlockchainConfig`] from the provided `network`
+///
+/// # Errors
+/// Returns `Err` if the network is not supported
+pub fn config(network: &str) -> Result<BlockchainConfig> {
+    // All available networks are listed here:
+    let (network, bip44_id, is_dev) = match network {
+        "dev" => ("dev", 1, true),
+        "goerli" => ("goerli", 1, true),
+        "mainnet" => ("mainnet", 42161, false),
```

Comment:
This bip44 id seems wrong, check arbitrum bip44 id here:
https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
resolved